test/e2e/onCLS-test.js

```diff
@@ -662,11 +662,14 @@ describe('onCLS()', async function () {
     if (!browserSupportsCLS) this.skip();
 
     await navigateTo('/test/cls?hidden=1');
+    // Wait for a frame to be painted.
+    await nextFrame();
```

Comment:
Do you know why the `nextFrame()` is needed here? Given that the page is loaded in the hidden state, I wonder if waiting for the load event would be better?

You could try `navigateTo('/test/cls?hidden=1', {readyState: 'complete'})` and see if that helps.

Reply:
Good idea, done!

Reply:
Actually let me see if that fixes the need for the `>=` mentioned above...